test/unit/utils.test.js

```diff
@@ -0,0 +1,5 @@
+describe("example test", () => {
```

Comment:
Did you mean to leave these empty tests?

Reply:
Yes! Placeholders for upcoming work to add tests. Arguably should've waited to include those files for when I start those tests (my next task), but don't see harm in having them here until then 🤷‍♀️